demos/demo_references.bib

```diff
@@ -25,6 +25,14 @@ @InBook{2015:lmscup
                   {http://www1.maths.leeds.ac.uk/~matak/lms-cup2015.pdf}
 }
 
+@Manual{Gidel:2017,
```

Comment:
Yes, but we still don't have volume and page information which are essential in the bibtex entry for articles.